using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -126,10 +118,111 @@ Set the Push Identifier request
 }
 ```
 
+### Analytics Identity Response
+
+#### Event Description
+
+This event is dispatched by the Analytics extension upon receiving the Analytics identity request event that gets generated in the Analytics [getTrackingIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#gettrackingidentifier) and [getVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#getvisitoridentifier-1) calls. This event contains the unique tracking identifier. This value may be null if identity extension is enabled and there is a valid ECID generated.
+
+#### Event Details
+
+| Type                          | Source                                 | Paired | Direction                                                    |
+| :---------------------------- | :------------------------------------- | :----- | :----------------------------------------------------------- |
+| com.adobe.eventType.analytics | com.adobe.eventSource.responseIdentity | True   | This is the end of the paired event Analytics Request Identity event containing the AID value |
+
+#### Event Data Payload Definition
+
+Definition of the key/value pairs that will be present in this event
+
+| Key  | Value Type | Optional | Description                             |
+| :--- | :--------- | :------- | :-------------------------------------- |
+| aid  | String     | No       | The analytics tracking identifier value |
+
+#### Event Data Example
+
+**sample**
+
+```
+{"aid" : "aidValue"}
+```
+
+### Audience Manager Content Response
+
+The Identity extension listens for the Audience extension response content event for any user opt-out updates. If this response contains `optedouthitsent: false` a new opt-out network call is sent to the Identity service.
+
+For the details about this event, please refer the [Audience Manager Event Reference document](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager/audience-manager-event-reference#audience-manager-content-response-1).
+
+### Configuration Response Content
+
+#### Event Description
+
+This event will be generated in the following scenarios:
```

Comment:
Similar here, these details should be covered in the Configuration event reference page and here we should just enumerate what are the keys required by the Identity extension.

Reply:
For Config, I am just leaving the details of the event as it is ... because https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-event-reference page is still to be added.